companies/swapfiets-be.json

```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "swapfiets-be",
+    "relevant-countries": [
+        "be"
+    ],
+    "name": "Swapfiets België BVBV",
+    "address": "Mechelseplein 10\n2000 Antwerpen\nBelgium",
```

Comment:
Can you add a source for that address? Or did I miss sth.?

Reply:
done